Jenkinsfile.kola.aws

```diff
@@ -55,6 +55,7 @@ podTemplate(cloud: 'openshift', label: 'coreos-assembler', yaml: pod, defaultCon
 
         stage('AWS Kola Run') {
           utils.shwrap("""
+          export AWS_CONFIG_FILE=\${AWS_FCOS_KOLA_BOT_CONFIG}
```

Comment:
Previously the AWS config was pre-validated by nature of the build having been uploaded with the credentials being used to test. With this change to using a different config we should validate that it exists before attempting this step.